### DISCUSSION TITLE
Customize relational interface buttons labels

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#12209] [Feature Requests]
There are times where we want to indicate a more precise action instead of `Create New`
For instance, if we add an evaluation, we would like to have a button `Evaluate`

It would be nice to be able to customize for some buttons `Create New` and  `Add existing` labels and have translations.
Also be able to change the color and add an icon.

**Comment by azrikahar** [comment]
Interesting! Seems like although Repeaters does provide an option to customize the `Create New` label:

![chrome_pc2NGP3pgj](https://user-images.githubusercontent.com/42867097/158795092-659ddfcb-5f05-4c8d-bc89-d895c8ad683e.png)

It is not present in relational interfaces like M2M, M2O etc. Just to confirm, are you using a particular relational interface and _not_ repeaters? (not recommending switching to repeater of course! just want to know for sure my interpretation is correct here)

#12170 may also be helpful to customize the translation, but in your case, having the option in the relational interfaces themselves may be the most customizable route.

As for color and icon, even though I understand that it's nice to have, we would also need to make sure not to add _too_ many minute permutations/customizations which may make things feel daunting to some users. But there's also the case of making it an `advanced` option as well. I'll defer to @benhaynes and @rijkvanzanten on this one 🙂

**Reply by Pachat** [reply]
Yes, you are right, it is with relational interfaces, at least O2M and M2M I use. (Precision added in the OP Title)

Thanks for the #12170 reference, but it cannot be applied here as we have no place to put a string (as in dropdowns' texts)
until the `"Create New" Label` as shown in your image becomes available for relational interface buttons.

I understand that - in second time features - colors and icons can make the basic administration UI heavier and would be better placed in an advanced option

**Reply by benhaynes** [reply]
I think we should remain consistent across all interface options, so I'd say we should add options to each (if applicable) for:

* "Create New" Label
* "Select Existing" Label

I think these should probably be considered "Advanced" (most users won't need to update them), so they won't make the normal field flow too busy.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
